### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mikeross0104**
- [DEMO LINK](https://mikeross0104.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://mikeross0104.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x]  Logo is an image wrapped with a link
- [x] CSS Variable is used for a blue color
- [x]  Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/mikeross0104/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#" class="logo-img"><img src="images/logo.png" alt="Moyo"></a>
```
Comment (Viktor-Kost) [inline src/index.html:19]:
```suggestion
      <a href="#" class="logo-img">
        <img src="images/logo.png" alt="Moyo">
      </a>
```

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+
```
Comment (Viktor-Kost) [inline src/index.html:17]: redundant
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:70]: move repeated colors to variables

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+  display: flex;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:28]: it's not a good practice to hardcode height for elements with content inside. Apply padding to your links

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,74 @@
...
+}
```
Comment (Viktor-Kost) [inline src/style.css:29]: redundant
```suggestion
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+  display: flex;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  align-items: center;
+  line-height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:42]: move this rule to a link and set correct value from figma (it's not 60px)

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,74 @@
...
+  text-decoration: none;
+  color: #000;
+  font-weight: 500;
+}
+
+.nav-item:first-child {
+  margin-left: 0;
+}
+
+.nav-item a:hover {
```
Comment (Viktor-Kost) [inline src/style.css:56]:
```suggestion
.nav-link:hover {
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Some moments can be improved

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please redeploy your solution 
image

**Pull request closed without merging**